### ISSUE TITLE
[Accessibility: ACC-270.1] Pop Up: UI not accessible using keyboard

### ISSUE THREAD

**Issue by divyamalhotra488** [SAP/fundamental-ngx#12583]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
Each & every UI, must be accessible using the keyboard.
Keyboard is the prerequisite for support of assistive technologies to enable users with disabilities to read and operate software applications, hence all the UIs must be accessible with keyboard.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Component used: [Form Message](https://sap.github.io/fundamental-ngx/#/core/form-message)
‘Error message’ pop ups are not accessible using the keyboard throughout the application.
<img width="452" alt="image" src="https://github.com/user-attachments/assets/1aa83702-f2af-4b71-8c26-442a75aa1b64">
<img width="452" alt="image" src="https://github.com/user-attachments/assets/3e4ad537-d907-4977-b3cb-e60c182094e6">
<img width="452" alt="image" src="https://github.com/user-attachments/assets/bba01c6b-8107-4ee9-9458-d12df34df505">

#### Please provide stackblitz example(s).
You can try ngx examples, issue is reproducible there.

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations.
Yes, issue is raised by an accessibility expert.

#### Did you check the documentation and the API?
Yes

#### Did you search for similar issues?
Yes

#### Is there anything else we should know?

**_IMPORTANT: Please refrain from providing links or screenshots of SAP's internal information, as this project is open-source, and its contents are accessible to anyone._**

**Comment by droshev** [comment]
@nikolay-kolarov Can you advise us in this question?

**Comment by nikolay-kolarov** [comment]
Hi @droshev,

Those are designed to be not focusable. Their text should be announced on focus or when the value state appears dynamically.
https://experience.sap.com/fiori-design-web/interaction-foundations-value-states/
https://ui5.sap.com/#/entity/sap.m.Input/sample/sap.m.sample.InputValueState

So I think you can close this one.

Thanks,
Nikolay
